### PR TITLE
mod: bump etcd to v3.5.7

### PR DESCRIPTION
**Pull request by joostjager**
Fix https://github.com/etcd-io/etcd/issues/14268

### CODE REVIEW

**Comment by joostjager** [comment]
Bumped to the next version 3.5.7 to see if that's better.

**Comment by guggero** [comment]
Doesn't seem to be the case. Do we need to create an issue in their repo? Sounds like something small.

**Comment by guggero** [comment]
Also, I think to avoid needing the replace, we instead should bump the dependencies correctly in the `kvdb/go.mod` file, merge, tag and then it will be bumped in the main mod file automatically when we go to the next `kvdb` tagged version.

**Comment by joostjager** [comment]
Created first step here: https://github.com/lightningnetwork/lnd/pull/7353

File: lncfg/db.go [hunk 1]
```diff
@@ -142,7 +143,7 @@ func (db *DB) Init(ctx context.Context, dbPath string) error {
 		db.Etcd = cfg
 
 	case db.Backend == PostgresBackend:
-		postgres.Init(db.Postgres.MaxConnections)
+		sqlbase.Init(db.Postgres.MaxConnections)
```
Comment (joostjager) [inline lncfg/db.go:146]: @ellemouton not sure if I am supposed to bump kvdb here before merge of #7252 ?

**Comment by joostjager** [comment]
@ellemouton fixed the final bump in 2fd4f3f180cecc54871515444491504d2ff732e9, closing.

**Pull request closed without merging**